doc/sample-data.md

```diff
@@ -0,0 +1,24 @@
+# Sample Data
+
+### Introduction
+**Intel® RealSense™ SDK 2.0** supports working with pre-recorded data ([learn more](https://github.com/IntelRealSense/librealsense/blob/master/src/media/readme.md))
+
+All files below can be opened using the [RealSense Viewer](https://github.com/IntelRealSense/librealsense/releases/download/v2.8.0/Intel.RealSense.Viewer.exe) (`Add Source` > `Load Recorded Sequence` or just drag & drop into the Viewer) 
+In addition, you can make any of the provided [examples](https://github.com/IntelRealSense/librealsense/tree/master/examples) run from file, by replacing `pipe.start()` with:
+```cpp
+rs2::config cfg;
+cfg.enable_device_from_file(<filename>);
+pipe.start(cfg); // Load from file
+```
+`*.bag` files contain uncompressed and unfiltered data and hence tend to be rather large (in the order of 100 MB per one second of recording) Our file-format is an extension of open [ROS-bag](http://wiki.ros.org/rosbag) format, meaning they can also be opened using existing robotics tools like [rqt-bag](http://wiki.ros.org/rqt_bag)
+
+
+> **Disclaimer**: The files below are not official marketing material. These are captured using pre-production hardware with the sole purpose of helping developers, makers and students get started with RealSense technology
+
+### Files
```

Comment:
Do those files target a specific usage? or do you plan to add more for specific usages?

Reply:
Not at the moment, just plain old data, to get people started. 
I hope more of our colleagues will get involved and provide better samples